BUTTON_IDEAS.md

```diff
@@ -10,7 +10,7 @@ When adding ideas, please add them in the right category (or add a new one if yo
 2. [Icon](#icon)
 3. [Special](#special)
 4. [Social Buttons](#socialButtons)
-
+5. [Dark Mode Toggle](#darkMode)
```

Comment:
Can you remove it from here and add it in the table under "Special" please?

Reply:
Sure.